setup.cfg

```diff
@@ -1,5 +1,5 @@
 [metadata]
-name = cloud_info_provider
+name = cloud_info
```

Comment:
@alvarolopez why?

Reply:
This is changed because:

- The name of the python package is actually `cloud_info` (since the root folder where all the code is located is called `cloud_info`)
- Setuptools, using the `setup.cfg` file, is going to install it as `cloud_info_provider`.

This will make things work, but it is not completely correct from my point of view (it may be confusing). I would expect that the python package name and the intsalled package name with setuptools remain the same. This can be done the other way round, renaming the directory and all the dot imports.

Reply:
OK, seems better indeed. I'm fine with this change, it will be easier to just change this.
May it cause problems to people having used pip to install it? Even if it's not really the supported/advised mean to deploy it, can we provide a solution for this? Or would mentioning it in the release notes be enough?